### PR TITLE
Added "successes story" html

### LINKED ISSUE #56: Html:Success story

### PR DESCRIPTION
**Pull request by firity**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,6 +64,32 @@ <h1 class="cTitle">300+</h1>
       </section>
 
       <section><!--sUCCESS STORY --></section>
+    <section class="suc-sto">
```
Comment (yildiztugba) [inline index.html]: Can you please delete success section with comment? Then i will merge it.

**Review by yildiztugba — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (yildiztugba) [reply index.html]: @firity sorry one more, did you format your code?

Comment (firity) [reply index.html]: Thank you for review.I didn't deleted but edit it,

**Review by yildiztugba — CHANGES_REQUESTED** [review]

**Comment by firity** [comment]
I really dont know how it is fomated ....can you please review this one and pull
thanks

**Pull request closed without merging**